### PR TITLE
Sheets snippets using Google Identity Services

### PR DESCRIPTION
**Pull request by RajeshGogo**
# Description
Sheets snippets using Google Identity Services and added exception handling.

Fixes # (230522680)

## Has it been tested?
- [X] Development testing done
- [ ] Unit or integration test implemented

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have performed a peer-reviewed with team member(s)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: adminSDK/directory/index.html [hunk 1]
```diff
@@ -57,7 +57,15 @@
        * Callback after api.js is loaded.
        */
       function gapiLoaded() {
+      try
+      {
         gapi.load('client', intializeGapiClient);
+        }
+      catch(ex)
```
Comment (sqrrrl) [inline adminSDK/directory/index.html:64]: Formatting still seems off.

Also, not sure if this try/catch is needed here. Don't think gapi.load is expected to throw anything, though could be wrong here. Under what conditions is an exception expected?

**Comment by sqrrrl** [comment]
Still can't merge due to unresolved comments. Besides formatting, not clear why all the try/catch statements are added and under what condition things are expected to fail. Most of those methods are async with callbacks and I don't think there's any condition where they would normally throw an error.

**Comment by sqrrrl** [comment]
Also need to resolve merge conflicts

**Comment by RajeshGogo** [comment]
Out of sync so closing PR

**Pull request closed without merging**